development/ui-kit/uikit.md

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
I think you need to leave this page and create an alias, so that the old page will point to the new page.